src/subclass/types.rs

```diff
@@ -363,95 +363,95 @@ pub trait ObjectSubclass: ObjectImpl + Sized + 'static {
     }
 }
 
-unsafe extern "C" fn class_init<T: ObjectSubclass>(klass: glib_sys::gpointer, _klass_data: glib_sys::gpointer)
```

Comment:
These should also all stay as separate functions. Too much code inline otherwise, they're basically like virtual methods above.